### PR TITLE
finished troubleshooting feature

### PR DESCRIPTION
**Pull request by nagalamadaka20p**

### CODE REVIEW

File: client/src/components/DataHome/DataLog/DataLog.jsx [hunk 1]
```diff
@@ -198,20 +198,6 @@ function DataLog({ handleSignInOpen, handleCreateAccOpen, isLoggedIn, id }) {
               onChange={(e) => onDrainColorChange(e)}
               value={drainColor}
             />
-            <h3>Drain output photo</h3>
```
Comment (dyi1) [inline client/src/components/DataHome/DataLog/DataLog.jsx:201]: From this change it looks like you're removing the ability for patients to upload picture. Is that an intended change?

File: client/src/components/Troubleshooting/Troubleshooting.jsx [hunk 2]
```diff
@@ -1,42 +1,379 @@
...
+              </select>
+            </div>
+          )}
+          {troubleshooting.persistentIncrease === "no" && (
+            <div className="q6">
+              <h2>Any changes in your health? Fever/pain?</h2>
+              <select
+                name="persistentincreasechange"
+                className="troubleshooting-dropdown"
+                value={troubleshooting.persistentincreasechange}
```
Comment (dyi1) [inline client/src/components/Troubleshooting/Troubleshooting.jsx:125]: Nit try to be consistent with your naming. I see you are using camel case through out most of it so hopefully we stay consistent on this?

**Review by dyi1 — COMMENTED** [review]
Something you can do when making these requests is to also take a video/screen shot of your live changes. That way, your reviewers can visually see what your changes will do. 

Think of it as a "how did you test that your changes work" sort of moment.

File: client/src/components/Troubleshooting/Troubleshooting.jsx [hunk 3]
```diff
@@ -1,42 +1,379 @@
...
+    };
+    newForm[key] = val;
+    setTroubleshooting(newForm);
+  }
 
-function Troubleshooting({handleSignInOpen, handleCreateAccOpen, isLoggedIn, troubleshooting, handleOnTroubleshootingChange}) {
   return (
-    <div className='troubleshooting'>
-        <NavBar
+    <div className="troubleshooting">
```
Comment (dyi1) [inline client/src/components/Troubleshooting/Troubleshooting.jsx:37]: truth be told I don't see an ez or quick way to make this faster. Looks like a rough change

File: client/src/components/DataHome/DataLog/DataLog.jsx (hunk 1 above)
Comment (nagalamadaka20p) [reply client/src/components/DataHome/DataLog/DataLog.jsx:201]: Yes I decided that I will work on this part after I finish the provider view and graph representation of data.

File: client/src/components/Troubleshooting/Troubleshooting.jsx (hunk 2 above)
Comment (nagalamadaka20p) [reply client/src/components/Troubleshooting/Troubleshooting.jsx:125]: Okay nice catch thank you!

**Pull request merged**